build.sbt

```diff
@@ -23,14 +25,9 @@ lazy val `hello-api` = (project in file("hello-api"))
 
 // ALL SETTINGS HERE ARE TEMPORARY WORKAROUNDS FOR KNOWN ISSUES OR WIP
 def workaroundSettings: Seq[sbt.Setting[_]] = Seq(
-  // This quickstart still doesn't use service discovery for the gRPC client
-  // so we hardcode the HTTPS port for the gRPC server to 11000
-  lagomServiceHttpsPort := 11000,
-  // There is a bug in akka-http 10.1.4 that makes it not work with gRPC+Lagom,
-  // so we need to downgrade to 10.1.3 (or move to 10.1.5 when that's out)
-  // https://github.com/akka/akka-http/issues/2168
-  dependencyOverrides += "com.typesafe.akka" %% "akka-http-core" % "10.1.3",
-  dependencyOverrides += "com.typesafe.akka" %% "akka-http" % "10.1.3"
+  // The certificate uses `localhost` as CN (Common Name) so that's the address we want the
+  // the service to be published under.
+  lagomServiceAddress := "localhost"
```

Comment:
1. Since it relies on akka-grpc `0.3+` there's no need to override akka-http.
1. ServiceDiscovery works IFF both descriptor and grpc use the same name.
1. service address _must_ be `localhost` for SSL certificate validation to pass.